### ISSUE TITLE
monotouch-test / iOS Unified 32-bits - simulator / Debug -> multiple failures

### ISSUE THREAD

**Issue by spouliot** [xamarin/xamarin-macios#11504]
### Steps to Reproduce

1. Start xharness 
2. Run "monotouch-test / iOS Unified 32-bits - simulator / Debug"

Sadly it seems the commit does not trigger this test suite to run (even if all of them should run, unless impossible - which is not the case here)

### Expected Behavior

No errors

### Actual Behavior

```
	[FAIL] TestCallbackPlayAlert :   TestCallbackPlayAlert
	[FAIL] TestCallbackPlaySystem :   TestCallbackPlaySystem
	[FAIL] EnumValues_22351 : AppleMoveTime could not be created: System.ArgumentNullException: Value cannot be null.
	[FAIL] ReturnReleaseTest : System.NullReferenceException : Object reference not set to an instance of an object
	[FAIL] TryCopyCurrentNetworkInfo :   Dictionary
Tests run: 2460 Passed: 2410 Inconclusive: 45 Failed: 5 Ignored: 342
```

### Environment

`main`

### Example

https://github.com/xamarin/xamarin-macios/pull/11374#issuecomment-837937044

PR with `run-all-tests` and `skip-device-tests`

**Comment by rolfbjarne** [comment]
@mandel-macaque is there a way to determine if the test run above occurred on a VM or if it was on real hardware?

**Comment by spouliot** [comment]
@rolfbjarne I could duplicate them locally (not on a VM)

**Comment by rolfbjarne** [comment]
* TestCallbackPlayAlert :  Fix in #11763.
* TestCallbackPlaySystem :  Fix in #11763.
* EnumValues_22351 : Fix in #11765.
* ReturnReleaseTest : Fixed in #11742.
* TryCopyCurrentNetworkInfo :   Fix in #11764.

**Comment by rolfbjarne** [comment]
This is fixed in `main` now.
